redis/google/cloud/redis_v1beta1/gapic/cloud_redis_client_config.py

```diff
@@ -17,24 +17,24 @@
                 }
             },
             "methods": {
-                "CreateInstance": {
+                "ListInstances": {
                     "timeout_millis": 60000,
                     "retry_codes_name": "non_idempotent",
```

Comment:
@busunkim96, @crwilcox Why have `ListInstances` and `GetInstance` been marked `non_idempotent` here?

Reply:
Looks to be intentional. This is the internal commit message:

> Replace non-standard retry parameter definitions for ListInstances and GetInstance.
>
> Replace LRO polling config.
>
> Removing these method configs meant the gapic-generator fell back on using the HTTP  Verb to assign a default retry config. The default values differed from what was specifically assigned to these two methods, thus the resulting GAPICs changed unexpectedly.

I think this is reverting part of https://github.com/googleapis/google-cloud-python/pull/9265/?